### PR TITLE
feat/is-blank

### PR DESCRIPTION
**Pull request by gijsvanwichen**

### CODE REVIEW

File: functions/is-blank/1.0/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const isBlank = async ({ record }) => {
```
Comment (thomasvanophem) [inline functions/is-blank/1.0/index.js:1]: I think we could write this as
```
const isBlank = async ({ record: { data } }) => ({
  result: !data
})
```

File: functions/is-blank/1.0/function.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "description": "Checks if the given record is blank and returns a Checkbox value that can be used in the condition step.",
+  "label": "Is blank",
+  "category": "Flow",
+  "icon": "CreateIcon",
```
Comment (thomasvanophem) [inline functions/is-blank/1.0/function.json:5]: We need a different icon for this function

**Review by thomasvanophem — APPROVED** [review]

File: functions/is-blank/1.0/function.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  "options": [{
+    "meta": {
+      "type": "Record",
+      "additional": {
+        "model": [
+          "name"
+        ]
+      },
```
Comment (thomasvanophem) [inline functions/is-blank/1.0/function.json:13]: Don't think we need the additional info in this case

**Comment by justin-betty** [comment]
Please add tests

**Comment by justin-betty** [comment]
Please add tests

**Comment by jwielinga** [comment]
Replaced by isPresent

**Pull request closed without merging**